### PR TITLE
REM-1110: node account supporting protobuf and changes

### PR DESCRIPTION
**Pull request by Alladin9393**
### Jira references
* Ticket (task) — https://remmeio.atlassian.net/browse/REM-1110

### CODE REVIEW

File: node_account.proto [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+message NodeAccountMethod {
+    enum Method {
+        TRANSFER_FROM_FROZEN_TO_UNFROZEN = 0;
+        TRANSFER_FROM_UNFROZEN_TO_OPERATIONAL = 1;
+    }
+}
+
+message NodeAccountInternalTransferPayload {
+    uint64 value = 1;
+}
```
Comment (eugene-babichenko) [inline node_account.proto]: Add trailing newline

File: node_account.proto [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+syntax = "proto3";
+
+message NodeAccount {
+    bool is_masternode = 1;
+    bool is_masternode_closed = 3;
```
Comment (eugene-babichenko) [inline node_account.proto]: Keep  consistnt numbering. 1 should be followed by 3.

File: node_account.proto [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    uint64 balance = 4;
+
+    message Reputational {
```
Comment (eugene-babichenko) [inline node_account.proto]: This message is unused.  If you intended to use it as a field, you need to instantiate it.

**Review by eugene-babichenko — CHANGES_REQUESTED** [review]
Please address the questions and document messages and their fields.

File: node_account.proto (hunk 3 above)
Comment (Alladin9393) [reply node_account.proto]: Something like this `repeated Reputational reputation = 3;`?

Comment (eugene-babichenko) [reply node_account.proto]: Why `repeated`? If this message is per node, then just a single field.

**Review by eugene-babichenko — APPROVED** [review]

**Review by artie-frants — APPROVED** [review]

**Comment by dmytrostriletskyi** [comment]
Do not merge until [#232](https://github.com/Remmeauth/remme-core/pull/232/files) isn't merged.

**Pull request merged**